src/common/walFilter/versions/recordProcessGPDB6.c

```diff
@@ -0,0 +1,503 @@
+#include "build.auto.h"
+
+#include <stdbool.h>
+#include "common/log.h"
+#include "postgres/interface/crc32.h"
+#include "recordProcessGPDB6.h"
+#include "xlogInfoGPDB6.h"
+
+#define XLR_INFO_MASK           0x0F
+#define XLOG_HEAP_OPMASK        0x70
+
+typedef uint32 CommandId;
+typedef uint16 OffsetNumber;
+typedef uint32 ForkNumber;
```

Comment:
I think it's worth returning the definition via enumeration, but without the incorrect comments inside it.